src/connectivities.jl

```diff
@@ -83,6 +83,11 @@ function connect(indices::Tuple, ::Type{Ngon})
   Connectivity{Ngon{N},N}(indices)
 end
 
+function connect(indices::Tuple, ::Type{Simplex})
+  N = length(indices)
+  Connectivity{Simplex{N-1},N}(indices)
+end
+
```

Comment:
I don't think this method is necessary given that we already have a general method for Polytope?